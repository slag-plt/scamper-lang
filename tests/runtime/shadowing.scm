(define x 3)

(define y (+ x 2))

(define x -5)

(+ x y)

(define f
  (lambda (x)
    (* x 2)))

(f 3)

(let
  ([z 10]
   [x (+ z x)]
   [z 100])
  (+ x z))

x