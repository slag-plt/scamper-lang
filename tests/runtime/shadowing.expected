0
6
105
-5
